packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -196,7 +201,7 @@ class DateTimePicker extends React.Component {
     // If the entered time (this.timeValue) is the missing hour during daylight savings,
     // it needs to be updated to the time in this.state.dateTime to reflect the change and force a render.
     if (this.state.dateTime && DateTimeUtils.isValidTime(this.timeValue, this.props.showSeconds)) {
-      const displayedTime = DateTimeUtils.getTime(this.state.dateTime.format(), this.props.showSeconds);
+      const displayedTime = DateTimeUtils.getTime(this.state.dateTime, this.props.showSeconds, this.props.timeZone);
```

Comment:
What is `.format()` removed from `this.state.dateTime` since `getTime` is expecting the first parameter to be an ISO string?

Reply:
Reverted to use .format();